index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+
+   <h1>Dr. Norman Borlaug</h1>
+   <div id="img-div">
+
+    <img src="dr.png" alt="" id="img">
+    <legend id="tribute-info">This is the picture of our doctor</legend>
+    <a href="" id="tribute-link" target="_blank"></a>
+
+   </div>
+
+
+
+
```

Comment:
- [ ] it's best practice to remove unwanted blank space in your code to make it easier to read , I recommend you to delete blank spaces from line 21 till 31.